chaos.proto

```diff
@@ -82,8 +82,34 @@ message Impact{
 
     //In POSIX time (i.e., number of seconds since January 1st 1970 00:00:00 UTC).
     optional uint64 notification_date = 9;
+
+    // Application pattern used while creating impact if any
+    repeated Pattern application_patterns = 10;
+}
+
+message WeekPattern {
+    optional bool monday = 1;
```

Comment:
minor: should you precise the default value in comment?

Reply:
Should'nt these be `required` instead of `optional` ? What does it means when the value is absent ?

Reply:
Done